CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Moves workflow engine to the same Kubernetes pod with the REANA-Job-Controller
+  (sidecar pattern).
+- Allows to set compute backend, name and Kerberos authentication requirement
+  for a job.
+- Allows partial workflow execution.
```

Comment:
Should we say some more?  ("...up to specific workflow step name")  Also, do we have examples in user docs? A quick grep in `reana-client` for `partial` did not return anything... Can you add some notes to `r-client` for each engine and then we can gently hint in the release notes here "... up to specific..." or somesuch?